package.json

```diff
@@ -15,8 +15,9 @@
     "vscode-languageclient": "5.2.1"
   },
   "devDependencies": {
+    "@types/azdata": "^1.40.0",
```

Comment:
Do you know why we're adding this NPM types module and including a copy of the azdata .d.ts file directly in the project.  Typically, you'd do one or other depending on if you need API changes that aren't included in the published module.

Reply:
On second look this looks fine.  It looks like you're removing the checked in azdata.d.ts and consuming from npm, but updating the proposed.d.ts file to get new interfaces.  This is the correct pattern that we should use here.

Reply:
This is from Charles' PR, not mine.